src/Automation/SnapshotCommand.cs

```diff
@@ -58,9 +62,6 @@ private static ScanResults ProcessResults(A11yElement element, Guid elementId, C
 
         private static OutputFile WriteOutputFiles(OutputFileFormat outputFileFormat, IScanTools scanTools, A11yElement element, Guid elementId)
         {
-            if (scanTools == null) throw new ArgumentNullException(nameof(scanTools));
```

Comment:
These checks now exist in the Execute method, so they never get triggered in this method